src/clr-core/package.js

```diff
@@ -21,7 +21,10 @@ const read = dir =>
       []
     );
 
-del.sync(['./dist/clr-core/**/*.{tsbuildinfo,ngsummary.json}', './dist/clr-core/*.{tsbuildinfo,ngsummary.json}']);
+del.sync([
+  './dist/clr-core/**/*.{tsbuildinfo,ngsummary.json,spec.js,spec.js.map,spec.d.ts}',
```

Comment:
This setup runs the unit tests side by side with the code we ship which should help reduce any build differences between tests and lib code. When we run a full build this deletes any test artifacts.